src/Layout/Presentation/ConnectedPresentation.tsx

```diff
@@ -28,7 +29,4 @@ const mapDispatchToProps = (dispatch: any): Partial<PresentationProps> => ({
     },
 });
 
-export const ConnectedPresentation = connect(
-    mapStateToProps,
-    mapDispatchToProps
-)(Presentation as any);
+export const ConnectedPresentation = connect(mapStateToProps, mapDispatchToProps)(Presentation as any);
```

Comment:
It really is a shame we aren't properly typing mapStateToProps and mapDispatchToProps (and still casting to any for connected components). Not much to do about it I guess at this stage.

Reply:
I'll make an issue for it. Perhaps we can get back to it later.